setup.py

```diff
@@ -29,15 +29,15 @@ def get_long_description() -> str:
 def get_install_requires() -> List[str]:
 
     requirements = [
-        "alembic",
+        "alembic>=1.5.0",
         "cliff",
         "cmaes>=0.8.2",
         "colorlog",
         "numpy",
         "packaging>=20.0",
         # TODO(kstoneriv3): remove this after deprecation of Python 3.6
         "scipy!=1.4.0" if sys.version[:3] == "3.6" else "scipy>=1.7.0",
-        "sqlalchemy>=1.1.0",
+        "sqlalchemy>=1.3.0",
```

Comment:
I changed the lower limit of version constraints for Alembic and SQLAlchemy for the following reasons:

* Alembic 1.5.1-1.8.1 is compatible with SQLAlchemy v1.3.0 or later.
* SQLAlchemy v1.3.0 is released at March 2019. According to [this page](https://pepy.tech/project/sqlalchemy?versions=1.3.*&versions=1.4.*&versions=1.2.*&versions=1.1.*), a few users have downloaded versions prior to v1.2.
* Both of alembic 1.5.0 and sqlalchemy 1.3.0 support Python 3.6.

Reply:
Thanks for sharing your thorough investigation report!

Reply:
Thanks for your comment. I also confirmed that `sqlalchemy==1.3.0` worked with `alembic==1.5.0`.